src/Tizen.NUI.Components/Style/SliderStyle.cs

```diff
@@ -240,90 +212,89 @@ public Extents TrackPadding
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <param name="bindableObject">The style that need to copy.</param>
+        /// <since_tizen> 8 </since_tizen>
         public override void CopyFrom(BindableObject bindableObject)
         {
             base.CopyFrom(bindableObject);
 
-            SliderStyle sliderAttributes = bindableObject as SliderStyle;
+            SliderStyle sliderStyle = bindableObject as SliderStyle;
 
-            if (null != sliderAttributes)
+            if (null != sliderStyle)
             {
-                if (sliderAttributes.Track != null)
+                if (sliderStyle.Track != null)
                 {
                     if (null == Track)
                     {
                         Track = new ImageViewStyle();
                     }
-                    Track.CopyFrom(sliderAttributes.Track);
+                    Track.CopyFrom(sliderStyle.Track);
                 }
 
-                if (sliderAttributes.Progress != null)
+                if (sliderStyle.Progress != null)
                 {
                     if (null == Progress)
                     {
                         Progress = new ImageViewStyle();
                     }
-                    Progress.CopyFrom(sliderAttributes.Progress);
+                    Progress.CopyFrom(sliderStyle.Progress);
                 }
 
-                if (sliderAttributes.Thumb != null)
+                if (sliderStyle.Thumb != null)
                 {
                     if (null == Thumb)
                     {
                         Thumb = new ImageViewStyle();
                     }
-                    Thumb.CopyFrom(sliderAttributes.Thumb);
+                    Thumb.CopyFrom(sliderStyle.Thumb);
                 }
 
-                if (sliderAttributes.LowIndicatorImage != null)
+                if (sliderStyle.LowIndicatorImage != null)
                 {
                     if (null == LowIndicatorImage)
                     {
                         LowIndicatorImage = new ImageViewStyle();
                     }
-                    LowIndicatorImage.CopyFrom(sliderAttributes.LowIndicatorImage);
+                    LowIndicatorImage.CopyFrom(sliderStyle.LowIndicatorImage);
                 }
 
-                if (sliderAttributes.HighIndicatorImage != null)
+                if (sliderStyle.HighIndicatorImage != null)
                 {
                     if (null == HighIndicatorImage)
                     {
                         HighIndicatorImage = new ImageViewStyle();
                     }
-                    HighIndicatorImage.CopyFrom(sliderAttributes.HighIndicatorImage);
+                    HighIndicatorImage.CopyFrom(sliderStyle.HighIndicatorImage);
                 }
 
-                if (sliderAttributes.LowIndicator != null)
+                if (sliderStyle.LowIndicator != null)
                 {
                     if (null == LowIndicator)
                     {
```

Comment:
I'm not sure why some Property are needed to initilaize but some are not.

In SwitchStyle, u didn't initalize property.
Could you explain about this?

Reply:
Actually LowIndicator Property and others will be initialized in InitSubStyle. They don't need to initialize in CopyFrom. I'll update these codes.